src/AuthenticationService.php

```diff
@@ -169,12 +169,8 @@ public function authenticate(ServerRequestInterface $request, ResponseInterface
         foreach ($this->authenticators() as $authenticator) {
             $result = $authenticator->authenticate($request, $response);
             if ($result->isValid()) {
-                if ($authenticator instanceof PersistenceInterface) {
```

Comment:
Currently only SessionAuthenticator implements PersistenceInterface. Probably the WIP CookieAuthenticator will implement it too.